src/classes/STG_InstallScript.cls

```diff
@@ -258,7 +258,7 @@ global without sharing class STG_InstallScript implements InstallHandler {
     * @return void
     */
     global static void disablePreferredEmailEnforcement(Version previousVersion) {
-        if(previousVersion != null && previousVersion.compareTo(new Version(1,31)) < 0) {
+        if(previousVersion != null && previousVersion.compareTo(new Version(1,33)) < 0) {
             UTIL_Debug.debug('***Upgrading to version 1.31');
```

Comment:
Minor thing, but the description above talks about version 1.30, and the debug comment below about version 1.31.